fake_test.go

```diff
@@ -307,7 +307,7 @@ func TestFakeCheck(t *testing.T) {
 	fake := NewFake(IPv4Family, "kube-proxy")
 
 	tx := fake.NewTransaction()
-	
```

Comment:
Oh huh, I guess my editor caught some stray tab characters. I can revert this if you like.